api-channel-jira/src/main/java/com/synopsys/integration/alert/api/channel/jira/distribution/custom/MessageReplacementValues.java

```diff
@@ -23,20 +23,8 @@
     private final String componentVersionName;
     private final String severity;
     private final String policyCategory;
-
-    public static MessageReplacementValues trivial(String providerLabel, String projectName) {
-        return new MessageReplacementValues(
-            providerLabel,
-            projectName,
-            MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE,
-            MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE,
-            MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE,
-            MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE,
-            MessageReplacementValues.DEFAULT_NOTIFICATION_REPLACEMENT_VALUE
-        );
-    }
-
-    public MessageReplacementValues(
+    
+    protected MessageReplacementValues(
```

Comment:
I would almost prefer that this be private and the builder was a `public static final` inner-class. Then this class could have a static initializer for the builder.